pkg/controller/core/localqueue_controller.go

```diff
@@ -105,34 +145,54 @@ func (r *LocalQueueReconciler) NotifyWorkloadUpdate(oldWl, newWl *kueue.Workload
 // +kubebuilder:rbac:groups=kueue.x-k8s.io,resources=localqueues/finalizers,verbs=update
 
 func (r *LocalQueueReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	var queueObj kueue.LocalQueue
```

Comment:
Please don't do drive-by var naming, the PR is already big, I'm ok with a follow up

Reply:
Reverted